speeduino/scheduler.h

```diff
@@ -117,167 +135,51 @@ struct Schedule {
   void (*EndFunction)();          ///< End Callback function for schedule
 
   //pure virtual functions, these are really defined in sub-classes
-  virtual COMPARE_TYPE getCounter(void)=0; //Function for getting counter value
-  virtual void setCompare(COMPARE_TYPE compareValue)=0; //Function for setting counter compare value
-  virtual void timerDisable(void)=0; //Function to disable timer for specific channel
-  virtual void timerEnable(void)=0; //Function to enable timer for specific channel
+  COMPARE_TYPE getCounter(void) { return counter; }; //Function for getting counter value
+  void setCompare(COMPARE_TYPE compareValue) { compare=(COMPARE_TYPE)compareValue); }; //Function for setting counter compare value
```

Comment:
```suggestion
  void setCompare(COMPARE_TYPE compareValue) { compare=(COMPARE_TYPE)compareValue; }; //Function for setting counter compare value
```
Just a typo there. Forgot to delete one bracket.